framework/sandstone.cpp

```diff
@@ -3814,7 +3822,7 @@ int main(int argc, char **argv)
     // a multi-socket system, it's done after the main run and only using the
     // failing tests.
     SandstoneApplication::PerCpuFailures per_cpu_failures;
-    vector<const struct test *> triage_tests;
+    std::vector<test_cfg_info> triage_tests;
```

Comment:
BTW, we need to remove `triage`, see #530

Reply:
Yup.

Reply:
Approved, merged, rebased on top of that.